docs/content/monitoring.md

```diff
@@ -89,6 +89,7 @@ When Prometheus is enabled in the status plugin (see [Configuration](../configur
 | last_success_bundle_download | gauge | Last successfully bundle download in UNIX nanoseconds.   | EXPERIMENTAL |
 | last_success_bundle_request | gauge | Last successfully bundle request in UNIX nanoseconds.    | EXPERIMENTAL |
 | bundle_loading_duration_ns | histogram | A histogram of duration for bundle loading.              | EXPERIMENTAL |
+| bundle_active_revision | gauge | First successfully bundle activation in UNIX nanoseconds. Contains the activeRevision label. | EXPERIMENTAL |
```

Comment:
Now that I read this, the other descriptions are a little off as well. "Last successfully bundle download" -> "Last successful bundle download" etc. -- Could you please adjust this?

Reply:
Done